test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/DefaultApplicationModelProviderTest.cs

```diff
@@ -727,10 +727,14 @@ public void BuildActionModels_MixedHttpVerbsAndRoutes_MultipleEmptyVerbsWithMult
             Assert.Equal(2, actions.Count());
 
             var action = Assert.Single(actions, a => a.AttributeRouteModel.Template == "Products");
-            Assert.Equal<string>(new string[] { "GET", "POST" }, action.HttpMethods);
+            Assert.Equal<string>(
```

Comment:
Leave comments whenever you sort for assertions so it's not undone by another dev.